src/Symfony/Component/String/AbstractString.php

```diff
@@ -438,6 +438,13 @@ public function kebab(): static
         return $this->snake()->replace('_', '-');
     }
 
+    public function pascal(): static
+    {
+        $this->string = ucfirst($this->camel());
```

Comment:
using `ucfirst` will not work properly for the unicode string implementation, as `ucfirst` does not handle unicode.

The fact that you add tests only for the ASCII case is a good proof that this is not handled.

Reply:
Hi @stof, you were right! I've replaced ucfirst with AbstractString::title and it seems to work nicely. Also added unicode test cases.

Thank you!